alerts/tuya.markdown

```diff
@@ -0,0 +1,10 @@
+---
+title: "Tuya: Delay in state syncing"
+created: 2021-12-15 00:00:00
+integrations:
+  - tuya
+homeassistant: ">2021.10.0"
+github_issue: https://github.com/home-assistant/core/issues/60034 (old)
+---
+
+Currently you might experience a delay in updating, or missing state updates when using the Tuya integration. That's the same issue warned one month ago.
```

Comment:
Please point me to the source of this being the same issue/source of it being confirmed the same issue.